src/dev.rs

```diff
@@ -66,7 +66,7 @@ pub enum VerifyFailure {
     /// A permutation did not preserve the original value of a cell.
     Permutation {
         /// The column in which this permutation is not satisfied.
-        column: Column<Any>,
+        column: metadata::Column,
```

Comment:
What is the motivation for this change?

Reply:
I want to be able to specify a column in an expected `VerifyFailure` error, without constructing an actual column: https://github.com/zcash/orchard/pull/101/files#diff-a61514f6b9ddb874831490698e7a84231d985315c900accb9c80c302dda52b9aR490